com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDDefaultSettings.cs

```diff
@@ -0,0 +1,708 @@
+using UnityEngine.Rendering;
+using System.Collections.Generic; //needed for list of Custom Post Processes injections
+using System.IO;
+using System.Diagnostics;
+#if UNITY_EDITOR
+using UnityEditorInternal;
+using UnityEditor;
+using UnityEditor.Rendering;
+using UnityEditor.Rendering.HighDefinition;
+#endif
+
+namespace UnityEngine.Rendering.HighDefinition
+{
+    enum ShaderVariantLogLevel
+    {
+        Disabled,
+        OnlyHDRPShaders,
+        AllShaders,
+    }
+
+    enum LensAttenuationMode
+    {
+        ImperfectLens,
+        PerfectLens
+    }
+
+    /// <summary>
+    /// High Definition Render Pipeline Default Settings.
+    /// Default settings are unique per Render Pipeline type. In HD, Default Settings contain:
+    /// - a default Volume (Global) combined with its Default Profile (defines which components are active by default)
+    /// - this Volume's profile
+    /// - the LookDev Volume Profile
+    /// - Frame Settings
+    /// - Various resources for runtime, editor-only, and raytracing
+    /// </summary>
+    public partial class HDDefaultSettings : RenderPipelineDefaultSettings
```

Comment:
Is this class supposed to be publicly accessible for our users?

If it's the case, please document all its public members or move them to internal

Reply:
i changed this to not be public anymore :)